server/crawlers/redditCrawler.js

```diff
@@ -8,7 +8,7 @@ async function redditCrawler(companyName) {
     refreshToken: process.env.Reddit_refreshToken,
   });
 
-  const subreddit = await scraper.getSubreddit("all");
+  const subreddit = await scraper.getSubreddit("business");
```

Comment:
Why business?

Reply:
I was curious what the results would be for different subreddits and forgot to change back, will fix!